src/style.css

```diff
@@ -1,3 +1,93 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
 body {
+  font-family: Roboto, Arial, sans-serif;
+  background-color: grey;
+  font-size: 12px;
+  font-weight: bold;
+  margin: 0;
+}
+
+.header {
+  width: 100%;
```

Comment:
It's better not to fix container size (like width or height) if there is no such requirement. Let the content size dictate it. This can help to avoid overflow or accidental scroll bar.